modules/persistent-storage-local-removing-devices.adoc

```diff
@@ -24,33 +19,72 @@ Deleting a persistent volume that is still in use can result in data loss or cor
 
 .Procedure
 
-. Edit the previously created local volume to remove any unwanted disks.
+To delete LVs or LVSs:
+
+. If there are any bound PVs, you need to delete the corresponding persistent volume claims (PVCs) to release the PVs:
+
+.. To find bound PVs, run the following command:
++
+[source, terminal]
+----
+$ oc get pv
+
+NAME   CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM                 STORAGECLASS   VOLUMEATTRIBUTESCLASS   REASON   AGE
+pv1    5Gi        RWO            Retain           Available                         slow           <unset>                          28s
+pv2    30Gi       RWX            Retain           Bound       openshift/storage     my-sc          <unset>                          168d
+----
++
+Bound PVs have a status of "Bound" and their corresponding PVCs appear in the "CLAIM" column. In the preceding example, the second PV is bound, and its PVC is "openshift/storage".
+
+.. Delete corresponding PVCs of bound PVs by running the following command:
++
+[source,terminal]
+----
+$ oc delete pvc <name>
+----
++ 
+In this example, you would delete PVC "openshift/storage".
+
+. If any PV owned by the LV or LVS has a "Retain" reclaim policy, back up any important data, and then delete the PV:
 
-.. Edit the cluster resource:
+.. To find PVs with "Retain" reclaim policy, run the following command:
 +
 [source,terminal]
 ----
-$ oc edit localvolume <name> -n openshift-local-storage
+$ oc get pv
+
+NAME                CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM   STORAGECLASS   REASON   AGE
+local-pv-1cec77cf   100Gi      RWO            Delete           Available           local-sc                88m
+local-pv-2ef7cd2a   100Gi      RWO            Delete           Available           local-sc                82m
+local-pv-3fa1c73    100Gi      RWO            Retain           Available           local-sc                48m]
 ----
```

Comment:
I am not sure about `Retain` policy. 
@dobsonj Could you help to double confirm how we process this `Retain` policy?